src/v2/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -44,7 +45,13 @@ const AuctionDetails: React.FC<AuctionDetailsProps> = ({ sale, me }) => {
       </GridColumns>
 
       <Spacer my={4} />
-
+      <Flex alignItems="center" justifyContent="space-between">
+        <Flex alignItems="center">
+          {!!sale.cascadingEndTimeInterval && (
+            <SaleDetailTimerFragmentContainer sale={sale} />
+          )}
+        </Flex>
+      </Flex>
```

Comment:
Can we get away with just 

```tsx
{!!sale.cascadingEndTimeInterval && (
  <SaleDetailTimerFragmentContainer sale={sale} />
```

with none of the flex containers? They don't seem necessary, or at least the second one can be omitted.

Reply:
Also, we'd want to put the guard on the outside so that if `cascadingEndTimeInterval` is falsy then the outer container don't render and embed empty divs

Reply:
things start getting a little wonky without the flex boxes 
![image](https://user-images.githubusercontent.com/8671643/159964548-b2dd590b-7a82-45f5-b29f-b508212923b9.png)
I'm sure there is a way around it, that being said front end is definitely not my forte.

Reply:
actually was able to get away with removing one.

Reply:
It might be worth adding `textAlign='left'` on the `<Text>` component itself, but if that doesn't work then all good 👍